src/lib/components/event/event-history-timeline.svelte

```diff
@@ -207,9 +262,15 @@
     visualizationRef;
 
   const drawTimeline = () => {
-    if (timeline) {
-      timeline.destroy();
```

Comment:
> The one thing that I noticed is if it's a Running workflow and you have auto-refresh on, it keeps shifting the spans to the right and the 'Fit' button doesn't work as expected.

@Alex-Tideman we may need that `timeline.destroy()` after all... There's some wonkiness with sizing going on with auto-refresh and `timeline.redraw()` doesn't seem to fix it 😞 

> What do you think about making all the rows this same shorter height? (the height of the row when there's no color span)

There is a "fixed" `groupHeightMode` option, unfortunately, it appears not to be working (looks like there is already an open issue for it). Another reason to maybe make our own timeline 😅 

> What do you think of linking to the child workflow in the title here? 

On it! Might need to fix some edge cases with navigation though (e.g. hitting the back button after navigating from an event group to child).